src/main/resources/templates/fragments/v2-entry-table.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
I don't think this file is actually used now you have removed the HTML view from V2?

Reply:
Yes, this should be removed